.github/workflows/SpellCheck.yml

```diff
@@ -0,0 +1,13 @@
+name: Spell Check
+
+on: [pull_request]
+
+jobs:
+  typos-check:
+    name: Spell Check with Typos
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Actions Repository
+        uses: actions/checkout@v3
+      - name: Check spelling
+        uses: crate-ci/typos@v1.16.23 
```

Comment:
Sadly `@v1` does not work: https://github.com/crate-ci/typos/issues/857

Maybe this would be better:
```
# https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
version: 2
updates:
  - package-ecosystem: "github-actions"
    directory: "/" # Location of package manifests
    schedule:
      interval: "weekly"
    ignore:
      - dependency-name: "typos"
        update-types: ["version-update:semver-patch"]
```
https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#specifying-dependencies-and-versions-to-ignore

Reply:
Well, I would argue that it would still be nice to get updates of the spell checker.

Reply:
You will, just not the patch releases, only the minor upgrades.
Given the frequency they upgrade, that will likely suffice?